File: src/org/ligi/gobandroid_hd/ui/fragments/DefaultGameExtrasFragment.java
package org.ligi.gobandroid_hd.ui.fragments;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

public class DefaultGameExtrasFragment extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        TextView tv = new TextView(this.getActivity());
        tv.setText("Implement me!");
        return tv;
    }

}

File: android/src/main/java/org/ligi/gobandroid_hd/ui/fragments/DefaultGameExtrasFragment.kt
package org.ligi.gobandroid_hd.ui.fragments

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.ViewGroup
import android.widget.TextView

class DefaultGameExtrasFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?)
            = TextView(this.activity).apply {
        text = "Implement me!"
    }

}
